src/constants/menus.js

```diff
@@ -26,13 +27,14 @@ const MENUS = {
   ],
   footer: [
     [
-      { text: 'Hacktoberfest', ...LINKS.hacktoberfest },
+      /*Remocing temporarily - justnems 20240301*/
+      /*{ text: 'Hacktoberfest', ...LINKS.hacktoberfest },*/
```

Comment:
I generally prefer to remove such dead code rather than keep it. It's always possible to look back at file commit history to quickly pull back such changes. Temporary removal typically finds a way to be permanent!

```suggestion
```